src/lib/data/dataview/dataview.ts

```diff
@@ -26,6 +27,14 @@ export class DataView extends WithEvents {
     });
   }
 
+  addFilter(widgetId: string, filter: Filter) {
+    this.dataSource.addFilter(widgetId, { [this.column]: filter });
+  }
+
+  removeFilter(widgetId: string) {
+    this.dataSource.removeFilter(widgetId);
+  }
+
```

Comment:
What about naming the parameter as `filterId` rather than `widgetId`? I think these filters will be used not just for widgets but other filters like polygons, circles, etc.

Reply:
Yes, it sounds good :)